front/app/modules/commercial/customizable_homepage_banner/admin/CTASettings/CustomizedButtonSettings.tsx

```diff
@@ -12,6 +12,16 @@ import { injectIntl, FormattedMessage } from 'utils/cl-intl';
 import { InjectedIntlProps } from 'react-intl';
 import useAppConfigurationLocales from 'hooks/useAppConfigurationLocales';
 import { isNilOrError } from 'utils/helperUtils';
+import styled from 'styled-components';
+
+const Container = styled(SectionField)`
+  margin-left: 28px;
```

Comment:
I would again be wary of adding margin to the container. It can be a real PITA if you need to reuse components elsewhere. This one will probably not be reused, but still we should make it a habit to not define it on containers.

Reply:
I agree. Let me move it